lib/db.js

```diff
@@ -83,29 +83,32 @@ module.exports = function (
 
   DB.prototype.createSessionToken = function (authToken, userAgentString) {
     log.trace({ op: 'DB.createSessionToken', uid: authToken && authToken.uid })
+    var sessionToken;
```

Comment:
I think this might be an artifact of some refactoring gone wrong, I can clean this up.